app/containers/message/Reply.js

```diff
@@ -159,25 +164,28 @@ const Reply = React.memo(({
 					attachment.description && styles.marginBottom,
 					{
 						backgroundColor: themes[theme].chatComponentBackground,
-						borderColor: themes[theme].borderColor
+						borderColor: attachment.color || themes[theme].borderColor
 					}
 				]}
 				background={Touchable.Ripple(themes[theme].bannerBackground)}
 			>
-				<View style={styles.attachmentContainer}>
-					<Title
-						attachment={attachment}
-						timeFormat={timeFormat}
-						theme={theme}
-					/>
-					<Description
-						attachment={attachment}
-						timeFormat={timeFormat}
-						getCustomEmoji={getCustomEmoji}
-						theme={theme}
-					/>
-					<Fields attachment={attachment} theme={theme} />
-				</View>
+				<>
+					<View style={[{ backgroundColor: attachment.color || 'transparent', opacity: 0.25 }, styles.attachmentBackground]} />
```

Comment:
Instead of adding this new transparent view, can you try https://github.com/ricokahler/color2k and calc the new hex with transparency?

Reply:
Sure, didn't know about this! Initially my approach was creating a function with all the hex values to convert to using the CSS/SVG specification with color names, but this looks more appealing.